<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KInvestmentView</class>
 <widget class="QWidget" name="KInvestmentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Investment Summary</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="m_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_equitiesTab">
      <attribute name="title">
       <string>Equities</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" stretch="70,250,300">
         <item>
          <widget class="QLabel" name="textLabel1">
           <property name="text">
            <string>Select Account:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KMyMoneyAccountCombo" name="m_accountComboBox"/>
         </item>
         <item>
          <spacer name="spacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>280</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QTreeView" name="m_equitiesTree">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Summary of the equities contained in this account, showing your holdings and their most recent price.</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::DoubleClicked</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_securitiesTab">
      <attribute name="title">
       <string>Securities</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout" name="m_securitiesLayout">
         <item>
          <widget class="QLineEdit" name="m_searchSecurities"/>
         </item>
         <item>
          <widget class="QTreeView" name="m_securitiesTree">
           <property name="editTriggers">
            <set>QAbstractItemView::DoubleClicked</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <property name="expandsOnDoubleClick">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QPushButton" name="m_editSecurityButton">
           <property name="toolTip">
            <string>Modify the selected entry</string>
           </property>
           <property name="whatsThis">
            <string>Change the security information of the selected entry.</string>
           </property>
           <property name="text">
            <string>Edit...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_deleteSecurityButton">
           <property name="toolTip">
            <string>Delete this entry</string>
           </property>
           <property name="whatsThis">
            <string>Remove this security item from the file</string>
           </property>
           <property name="text">
            <string>Delete...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>110</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="spacer7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>51</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyAccountCombo</class>
   <extends>KComboBox</extends>
   <header>kmymoneyaccountcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
